hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/profiling/ConsumerProfiler.java

```diff
@@ -0,0 +1,16 @@
+package pl.allegro.tech.hermes.consumers.consumer.profiling;
+
+public interface ConsumerProfiler {
+
+    void startMeasurements(Measurement measurement);
+
+    void measure(Measurement measurement);
+
+    void startPartialMeasurement(Measurement measurement);
```

Comment:
For me it is not clear what is the difference between `measurement` and `partialMeasurement`, it is also not clear why one should be stopped and the other not. Can we document it?